test/ossl_shim/ossl_shim.cc

```diff
@@ -968,7 +968,7 @@ static bool DoExchange(bssl::UniquePtr<SSL_SESSION> *out_session,
   }
   if (config->enable_all_curves) {
     static const int kAllCurves[] = {
-      NID_X9_62_prime256v1, NID_secp384r1, NID_secp521r1, NID_X25519,
+      NID_X9_62_prime256v1, NID_secp384r1, NID_secp521r1, NID_X25519, NID_X448
```

Comment:
If only to avoid questions like this one, should this be in the same order as ```eccurves_default```?

Reply:
Ok - done.